### PR TITLE
wrangler 1.2

### PR DESCRIPTION
**Pull request by ilovezfs**
upstream fix for erlang 19
add a head spec
reparallelize the build
add a test

### CODE REVIEW

**Comment by ilovezfs** [comment]
Version in core is currently broken due to erlang 19 (and would need same patch as here), which was not noticed during https://github.com/Homebrew/homebrew-core/pull/2279 apparently.

**Comment by DomT4** [comment]
Yeah, this had no test so would have been completely ignored by CI. AFAIK `erlang` also doesn't ship any dynamic libraries, so `linkage` wouldn't have spotted anything off either.

File: Formula/wrangler.rb [hunk 1]
```diff
@@ -13,9 +23,13 @@ class Wrangler < Formula
...
   def install
-    ENV.deparallelize
     system "./configure", "--prefix=#{prefix}"
     system "make"
     system "make", "install"
   end
+
+  test do
+    suffixtree = Dir.glob("#{lib}/erlang/*/*/*/suffixtree").shift
+    assert File.executable?(suffixtree), "suffixtree must be executable"
```
Comment (MikeMcQuaid) [inline Formula/wrangler.rb]: Do we have anything we can run (even for a `--version`)? I fear this won't detect breakage.

Comment (ilovezfs) [inline Formula/wrangler.rb]: none that I could find, or it would be here!

**Comment by ilovezfs** [comment]
I think erlang stuff may need to be revision bumped always until the new version stuff is in place because my general understanding is the builds are highly likely to break unless upstream has already deliberately adapted the build to the latest. In this I think this thing is mainly emacs bindings? But I wouldn't be surprised if there are other broken erlang builds right now.

**Comment by ilovezfs** [comment]
Yeah, erlang 19 also broke disco, and yaws has a checksum mismatch but their current tarball builds.

**Comment by ilovezfs** [comment]
Time for disco to die I think: https://github.com/Homebrew/homebrew-boneyard/pull/141.

**Comment by ilovezfs** [comment]
Yaws fix: https://github.com/Homebrew/homebrew-core/pull/3794.

**Comment by apjanke** [comment]
I think this is ready to merge?

**Comment by ilovezfs** [comment]
@apjanke assuming objections to awful test have been dropped ...

**Comment by ilovezfs** [comment]
@apjanke :ship:'d.

**Pull request closed without merging**